netjsonconfig/backends/openwrt/converters/radios.py

```diff
@@ -55,6 +58,26 @@ def __intermediate_hwmode(self, radio):
         else:
             return '11a'
 
+    def __intermediate_band(self, radio):
+        """
+        possible return values are: 2g, 5g, 6g, 60g
+        """
+        protocol = radio['protocol']
+        radio.pop('hwmode', None)
+        if protocol in ['802.11b', '802.11g']:
+            return '2g'  # 2GHz
+        if protocol == '802.11ax' and radio['channel'] % 4 == 1:
+            return '6g'  # 6GHz
```

Comment:
Is this your observation or is this a convention?

Reply:
By convention, both `5g` and `6g` band uses `802.11ax` protocol.
And the `radio['channel'] % 4 == 1` is the observation:
1. OpenWrt docs mentions to refer wikipedia https://openwrt.org/docs/guide-user/network/wifi/basic#wi-fi_devices
2. By observation https://en.wikipedia.org/wiki/List_of_WLAN_channels#6_GHz_(802.11ax), it is clear that `6g` channel is 1 more than channel of `5g`.

Reply:
There are some exceptions to this observation. In 5 GHz 802.11ax, there are some channels that fulfil `radio['channel'] % 4 == 1` criteria. Search for channel 149, 157, 161, etc.

Reply:
I observed that, but what to do in overlapping cases (For now, in https://github.com/openwisp/netjsonconfig/pull/236/commits/00a697dbae67b0aa0dc1b4d1bc89086bbd58db80 I have given priority to 5ghz band)?